lib/src/migrators/module.dart

```diff
@@ -522,45 +522,46 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
   /// Adds a namespace to any function call that requires it.
   @override
   void visitFunctionExpression(FunctionExpression node) {
-    super.visitFunctionExpression(node);
-    if (node.namespace != null) return;
-    if (references.sources.containsKey(node)) {
-      var declaration = references.functions[node];
-      _unreferencable.check(declaration, node);
-      _renameReference(nameSpan(node), declaration);
-      _patchNamespaceForFunction(node, declaration, (namespace) {
-        addPatch(patchBefore(node.name, '$namespace.'));
-      });
-    }
+    if (node.namespace == null) {
```

Comment:
Rather than fully re-indenting everything, I'd write:

```dart
if (node.namespace != null) {
  super.visitFunctionExpression(node);
  return;
}
// ...
```

I think the one duplicated line is better than the many indented lines.

Reply:
Done